internal/blobtesting/map.go

```diff
@@ -100,7 +115,13 @@ func (s *mapStorage) PutBlob(ctx context.Context, id blob.ID, data blob.Bytes, o
 
 	data.WriteTo(&b)
 
+	if s.limit >= 0 && s.totalBytes+int64(b.Len()) > s.limit {
```

Comment:
```suggestion
	if s.limit >= 0 && s.totalBytes+int64(b.Len()) >= s.limit {
```

Reply:
i don't think this is correct, map of size 10 can fit 10 bytes, not 9.

Reply:
Ah, I see this check is with the incoming blob length. Makes sense.